src/components/Wishlist/Wishlist.js

```diff
@@ -0,0 +1,33 @@
+import { WishlistHeader, WishlistSection } from './index'
+import { Main, Text } from '../Reusable/index'
+import { getBgColor } from '../../utils/theme.util'
+import { useTheme } from '../../contexts/theme.context'
+
+const Wishlist = () => {
+    const { theme } = useTheme()
+
+    return (
+        <div
+            style={{
+                minHeight: '100vh'
+            }}
```

Comment:
Try to avoid inline styling.

Reply:
actually i want this to have priority over anything else so that is why i have given the `min-height` as inline.